kinto/core/cache/__init__.py

```diff
@@ -92,5 +94,7 @@ def ping(request):
         except:
             logger.exception("Heartbeat Failure")
             return False
+        finally:
+            transaction.abort()
```

Comment:
Why not... it will leave a useless trace in the DB, but we delete it on each call anyway.
Your call!